oauth2client/flask_util.py

```diff
@@ -331,6 +333,20 @@ def _make_flow(self, return_url=None, **kwargs):
             redirect_uri=url_for('oauth2.callback', _external=True),
             **kw)
 
+        flow_key = 'google_oauth2_flow_{0}'.format(csrf_token)
+        session[flow_key] = pickle.dumps(flow)
+
+        return flow
+
+    def _get_flow_for_token(self, csrf_token):
```

Comment:
Huh; I didn't know that. But that's an implementation pattern that we needn't and shouldn't follow, no matter who else does it.